treeherder/config/settings.py

```diff
@@ -113,11 +113,11 @@
 #
 # which django-environ converts into the Django DB settings dict format.
 DATABASES = {
-    'default': env.db_url('DATABASE_URL'),
+    'default': env.db_url('DATABASE_URL',default='DEBUG'),
```

Comment:
In general, it's *very* important you pay close attention to the details 

Please carefully read the values for each env variable:
https://github.com/mozilla/treeherder/issues/5926#issue-559802976

Defaulting every value to DEBUG is incorrect.

Reply:
Apologies sir. Let me correct it!